service/service_windows.go

```diff
@@ -122,16 +122,16 @@ const couldNotConnect syscall.Errno = 1063
 // On non-windows platforms, the function does nothing. The
 // stopCallback function is called when the Stop/Shutdown
 // request is received.
-func ProcessWindowsControlEvents(stopCallback func()) {
+func ProcessWindowsControlEvents(logger *logp.Logger, stopCallback func()) {
 	defer close(serviceInstance.executeFinished)
 
 	//nolint:staticcheck // keep using the deprecated method in order to maintain the existing behavior
 	isInteractive, err := svc.IsAnInteractiveSession()
 	if err != nil {
-		logp.Err("IsAnInteractiveSession: %v", err)
+		logger.Errorf("IsAnInteractiveSession: %v", err)
 		return
 	}
-	logp.Debug("service", "Windows is interactive: %v", isInteractive)
+	logger.Debugf("service", "Windows is interactive: %v", isInteractive)
```

Comment:
The equivalent to this is
```suggestion
	logger.Named("service")..Debugf("Windows is interactive: %v", isInteractive)
```